packages/block-library/src/table/edit.js

```diff
@@ -429,21 +428,19 @@ function TableEdit( {
 	return (
 		<figure { ...useBlockProps() }>
 			{ ! isEmpty && (
-				<BlockControls>
-					<ToolbarGroup>
```

Comment:
As far as I understand, `BlockControlSlot` already returns a `ToolbarGroup` and should not be necessary anymore.